### ISSUE TITLE
[v1.0.0-rc5] index.ts: [BABEL]: Cannot find module 'expo/config'

### ISSUE THREAD

**Issue by MartinHarkins** [expo/router#270]
### Summary

I'm getting this error when the app is loading on mobile:
```
Android Bundling failed 294ms
error: index.ts: [BABEL]: Cannot find module 'expo/config'
Require stack:
- /home/martin/code/myteamlink/mtl/node_modules/expo-router/node/getExpoConstantsManifest.js
- /home/martin/code/myteamlink/mtl/node_modules/expo-router/babel.js
- /home/martin/code/myteamlink/mtl/node_modules/@babel/core/lib/config/files/module-types.js
- /home/martin/code/myteamlink/mtl/node_modules/@babel/core/lib/config/files/configuration.js
- /home/martin/code/myteamlink/mtl/node_modules/@babel/core/lib/config/files/index.js
- /home/martin/code/myteamlink/mtl/node_modules/@babel/core/lib/index.js
- /home/martin/code/myteamlink/mtl/node_modules/metro-transform-worker/src/index.js
- /home/martin/code/myteamlink/mtl/node_modules/metro/src/DeltaBundler/Worker.flow.js
- /home/martin/code/myteamlink/mtl/node_modules/metro/src/DeltaBundler/Worker.js
- /home/martin/code/myteamlink/mtl/node_modules/jest-worker/build/workers/processChild.js (While processing: /home/martin/code/myteamlink/mtl/node_modules/expo-router/babel.js)
```

I can fix it by editing the import in `node_modules/expo-router/node/getExpoConstantsManifest.js` =>

```diff
- const { getConfig, getNameFromConfig } = require("expo/config");
+ const { getConfig, getNameFromConfig } = require("@expo/config");
```

I'm not 100% certain as to when this error showed up, but likely it was either when I integrated the expo app into a NextJS app or added in [solito](https://github.com/nandorojo/solito).

I couldn't find a reference for the package *expo/config*, though obviously there is *@expo/config*.

Thanks !

### Minimal reproducible example

I'll try to supply one if you believe you need one. However it seems to me that it's just an invalid import bug that's gone through the cracks.
Basically you should be able to hit it on any project with the env variable `APP_MANIFEST` unset.

**Comment by EvanBacon** [comment]
Install `expo@46.0.13` or greater.
